src/TagsInput/TagsInput.js

```diff
@@ -140,15 +140,11 @@ export default class TagsInput extends React.Component<Props, State> {
     return (
       <TouchableWithoutFeedback onPress={this.handleOnFocus}>
         <View style={[styles.container, dynamicStyle.border]}>
-          {Boolean(label) && (
+          {label != null && label !== '' && (
```

Comment:
Does flow complain about `Boolean(label)`?

Reply:
```
Error ┈┈┈┈┈┈┈┈┈ src/TagsInput/TagsInput.js:145:16

Cannot create Text element because in property
children:
 • Either undefined [1] is incompatible with
   null [2].
 • Or property @@iterator is missing in
   undefined [1] but exists in $Iterable [3].

     src/TagsInput/TagsInput.js
 [1]  23│   +label?: string,
        :
     141│       <TouchableWithoutFeedback onPress={this.handleOnFocus}>
     142│         <View style={[styles.container, dynamicStyle.border]}>
     143│           {Boolean(label) && (
     144│             <Text weight="bold" style={[styles.label, dynamicStyle.label]}>
     145│               {label}
     146│             </Text>
     147│           )}
     148│           <ScrollView
     149│             contentContainerStyle={styles.scrollContainer}

     /private/tmp/flow/flowlib_3e56be5a/react.js
 [2]  14│   | null
        :
 [3]  20│   | Iterable<?React$Node>;
```